client/packages/programs/src/JsonForms/common/JsonForm.tsx

```diff
@@ -259,6 +262,9 @@ export const JsonForm: FC<
           textAlign: 'right',
           whiteSpace: 'nowrap',
         },
+        '&:empty': {
+          display: 'none',
+        },
```

Comment:
Now that content could be rendered into toolbar portal, this div actually might end up empty. It shouldn't just take up a whole bunch of space if it is 😁